app/src/Talk/TalkController.php

```diff
@@ -444,6 +446,34 @@ public function reportComment($eventSlug, $talkSlug, $commentHash)
         $this->application->redirect($url);
     }
 
+    public function unlinkSpeaker($eventSlug, $talkSlug, $username)
+    {
+        if (!isset($_SESSION['user'])) {
+            $thisUrl = $this->application->urlFor('talk', ['eventSlug' => $eventSlug, 'talkSlug' => $talkSlug]);
```

Comment:
I'd move this out of the if-clause to be able to use it as redirection URL after the action is finished.